### ISSUE TITLE
gRPC health check endpoint for ambassador

### ISSUE THREAD

**Issue by Sushma10037017** [datawire/ambassador#3276]
Hi,
We have ambassador api-gateway(v1.2.2) deployed in our Kubernetes cluster and exposed via AWS application load balancer.
We want to support gRPC services routing via ambassador. AWS ALB supports gRPC flow but expects the health checks to return gRPC status code.

Can anybody help us to know if the ambassador exposes gRPC health check endpoint(We tried healthcheck endpoints mentioned [here](https://github.com/datawire/ambassador/blob/master/api/envoy/api/v2/core/health_check.proto#L155) )? Any recommendations around this are appreciated.

As a hack round for this, we are planning to deploy a sidecar/deployment which will be a gRPC service that checks ambassador REST health check endpoint. 
Thanks!.

**Comment by kflynn** [comment]
I don't think that any gRPC health check is exposed. If you're thinking about whipping up a gRPC health check service, though, might it be simpler to just add that directly to `cmd/entrypoint/healthcheck_server.go`? That's where the health checks are managed in modern Ambassador; we'd be delighted to support that effort... 🙂
